From c009e034fdd47d59889072795560ce65a1989deb Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 11:12:31 +0300
Subject: [PATCH] Makefile: add more repos to gaudi/dali-release

Add all the embedded related projects to gaudi/dali-release target.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 19ce7f5..c2de81f 100644
--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,8 @@
 	$(error HL_RELEASE_VER is undefined)
 endif
 
-RELEASE_REPOS := u-boot linux
+#TODO: add btl once it is separated from br-ext
+RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot
 
 .PHONY: gaudi-release dali-release
 gaudi-release dali-release: hl-check-release
